iree/compiler/Codegen/SPIRV/SPIRVLowerExecutableTargetPass.cpp

```diff
@@ -111,7 +111,7 @@ void SPIRVLowerExecutableTargetPass::runOnOperation() {
         addSPIRVTileAndVectorizeToCooperativeOpsPassPipeline(nestedModulePM);
         break;
       default:
-        llvm_unreachable("Unsupported pipeline on GPU target.");
```

Comment:
Coulld you make this

variantOp.emitOpError("unsupported pipeline on SPIRV target");
return signalPassFailure();

Reply:
done